docs/usage/web-api.md

```diff
@@ -110,133 +110,13 @@ requiring some authentication and compensation for the funds used.
-- `https://walrus-testnet.blockscope.net`
-- `https://walrus-testnet-aggregator.nodes.guru`
-- `https://walrus-cache-testnet.overclock.run`
-- `https://walrus-testnet-aggregator.stakin-nodes.com`
-- `https://testnet-aggregator-walrus.kiliglab.io`
-- `https://walrus-cache-testnet.latitude-sui.com`
-- `https://walrus-testnet-aggregator.nodeinfra.com`
-- `https://walrus-testnet-aggregator.stakingdefenseleague.com`
-- `https://walrus-aggregator.rubynodes.io`
-- `https://walrus-testnet-aggregator.brightlystake.com`
-- `https://walrus-testnet-aggregator.nami.cloud`
-- `https://aggregator.testnet.walrus.mirai.cloud`
-- `https://walrus-testnet-aggregator.stakecraft.com`
-- `https://agg.test.walrus.eosusa.io`
-- `https://walrus-agg.testnet.obelisk.sh`
-- `https://walrus-test-aggregator.thepassivetrust.com`
-- `https://walrus-testnet-aggregator.natsai.xyz`
-- `https://walrus.testnet.aggregator.stakepool.dev.br`
-- `https://aggregator.walrus.banansen.dev`
-- `https://testnet.aggregator.walrus.silentvalidator.com`
-- `https://testnet-aggregator.walrus.graphyte.dev`
-- `https://walrus-testnet-aggregator.imperator.co`
-- `https://walrus-testnet-aggregator.unemployedstake.co.uk`
-- `https://aggregator.walrus-01.tududes.com`
-- `https://walrus-aggregator-testnet.n1stake.com`
-- `https://suiftly-testnet-agg.mhax.io`
-- `https://walrus-testnet-aggregator.trusted-point.com`
-- `https://walrus-testnet-aggregator.veera.com`
-- `https://aggregator.testnet.walrus.atalma.io`
-- `https://153-gb3-val-walrus-aggregator.stakesquid.com`
-- `https://sui-walrus-testnet.bwarelabs.com/aggregator`
-- `https://walrus-testnet.chainbase.online/aggregator`
-- `https://walrus-tn.juicystake.io:9443`
-- `https://walrus-agg-testnet.chainode.tech:9002`
-- `https://walrus-testnet-aggregator.starduststaking.com`
-- `https://walrus-aggregator-testnet.cetus.zone`
-- `https://walrus-testnet-aggregator.everstake.one`
-- `http://walrus.testnet.pops.one:9000`
-- `http://scarlet-brussels-376c2.walrus.bdnodes.net:9000`
-- `http://aggregator.testnet.sui.rpcpool.com:9000`
-- `http://walrus.krates.ai:9000`
-- `http://walrus.globalstake.io:9000`
-- `http://walrus-testnet.staking4all.org:9000`
-- `http://walrus-testnet.rpc101.org:9000`
-- `http://93.115.27.108:9000`
-- `http://65.21.139.112:9000`
-- `http://162.19.18.19:9000`
-- `http://walrus-aggregator.stakeme.pro:9000`
-- `http://walrus-storage.testnet.nelrann.org:9000`
-- `http://walrus-testnet.senseinode.com:9000`
-- `http://walrus-testnet.equinoxdao.xyz:9000`
-- `https://walrus-testnet-aggregator.stakely.io`
-- `https://walrus-testnet-aggregator.talentum.id`
-- `http://37.27.230.228:9000`
-- `https://walrus-testnet-aggregator.luckyresearch.org/`
-- `http://walrus-testnet.suicore.com:9000`
-- `https://walrus-testnet.validators.services.kyve.network/aggregate`
-- `https://walrus-aggregator-testnet.suisec.tech`
-- `https://walrus-testnet-aggregator.dzdaic.com`
+{{ #aggregators }}
+{{ /aggregators }}
```

Comment:
Two questions:
1. Can we sort them in some way? It seems they are sorted alphabetically by default, but I'd prefer to have those with HTTPS first and those with bare IP addresses last.
2. Can we somehow indicate additional info like whether they have a cache?